[TypeScript] Duplicate definitions of `freeTrialPeriodAndroid`

### Version of react-native-iap

2.4.0-beta6

### Version of react-native

Any

### Platforms you faced the error (IOS or Android or both?)

Both

### Details

There are two definitions of `freeTrialPeriodAndroid` under Subscription in [index.d.ts](https://github.com/dooboolab/react-native-iap/blob/master/index.d.ts#L32). One being an optional string, the other a required one, which renders the following errors:

```
node_modules/react-native-iap/index.d.ts(28,3): error TS2300: Duplicate identifier 'freeTrialPeriodAndroid'.
node_modules/react-native-iap/index.d.ts(28,3): error TS2687: All declarations of 'freeTrialPeriodAndroid' must have identical modifiers.
node_modules/react-native-iap/index.d.ts(32,3): error TS2300: Duplicate identifier 'freeTrialPeriodAndroid'.
node_modules/react-native-iap/index.d.ts(32,3): error TS2687: All declarations of 'freeTrialPeriodAndroid' must have identical modifiers.
node_modules/react-native-iap/index.d.ts(32,3): error TS2717: Subsequent property declarations must have the same type.  Property 'freeTrialPeriodAndroid' must be of type 'string | undefined', but here has type 'string'.
```

### Solution

Remove one of the definitions.

--- a/index.d.ts
+++ b/index.d.ts
@@ -25,7 +25,6 @@ export interface Subscription<ID extends string> extends Common {
   subscriptionPeriodNumberIOS?: string
   subscriptionPeriodUnitIOS?: number
 
-  freeTrialPeriodAndroid?: string
   introductoryPriceCyclesAndroid?: number
   introductoryPricePeriodAndroid?: string
   subscriptionPeriodAndroid?: string

